x-pack/plugins/security_solution/public/common/components/event_details/related_cases.tsx

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo, useState, useEffect } from 'react';
+import { EuiFlexItem, EuiLoadingContent } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '../../../common/lib/kibana';
+import { TimelineEventsDetailsItem } from '../../../../common/search_strategy';
+import { getFieldValue } from '../../../detections/components/host_isolation/helpers';
+import { CaseDetailsLink } from '../links';
+
+interface Props {
+  data: TimelineEventsDetailsItem[];
+  isAlert: boolean;
+}
+
+type RelatedCaseList = Array<{ id: string; title: string }>;
+
+export const RelatedCases: React.FC<Props> = React.memo(({ data, isAlert }) => {
+  const {
+    services: { cases },
+  } = useKibana();
+
+  const alertId = useMemo(() => getFieldValue({ category: '_id', field: '_id' }, data), [data]);
+
+  const [relatedCases, setRelatedCases] = useState<RelatedCaseList>([]);
+  const [areCasesLoading, setAreCasesLoading] = useState(true);
+
+  const getRelatedCases = useCallback(async () => {
+    let relatedCaseList: RelatedCaseList = [];
+    try {
+      const relatedCasesResponse = await cases.api.getRelatedCases(alertId, {
+        owner: 'securitySolution',
```

Comment:
I think there is a constant for that

Reply:
Aha, just replaced it with the APP_ID. Thanks!